docs/guides/aws-private-link-workspace.md

```diff
@@ -289,7 +289,8 @@ resource "databricks_mws_workspaces" "this" {
   account_id                 = var.databricks_account_id
   aws_region                 = var.region
   workspace_name             = local.prefix
-  deployment_name            = local.prefix
+  # only set this if you have worked with databricks to set a deployment name
```

Comment:
probably rename this field from HCL example and add this sentence in the attribute reference in docs/resources/mws_workspaces.md.